compliance_checker/tests/data/non-comp/bad.cdl

```diff
@@ -45,10 +45,7 @@ variables:
 		lev2:standard_name = "atmosphere_hybrid_height_coordinate" ;
 		lev2:formula_terms = "a: var1 b: var2 orog: var3" ;
 		lev2:description = "Missing variables from formula_terms" ;
-	double bad_time_1(time) ;
-		bad_time_1:standard_name = "time" ;
-		bad_time_1:description = "lacks units" ;
-	double bad_time_2(time) ;
+	double time(time) ;
 		bad_time_2:units = "seconds" ;
```

Comment:
i think you need to rename these prefixes as well.  also this likely breaks the test changed below (the assert will not have the test `bad_time_2` in it anymore)